docs/src/frontend/index.md

```diff
@@ -696,6 +696,26 @@ acc.tokensAccepted() => Promise<Array<Token>>
 Returns a Promise for an array of tokens that are accepted by `acc`.
 On networks which do not keep track of this information (e.g. Ethereum), this returns an empty array.
 
+---
+@{ref("js", "appOptIn")}@{ref("js", "stdlib.accountAppOptIn")}
+```js
+acc.appOptIn(contract) => Promise<void>
+stdlib.accountAppOptIn(account, contract) => Promise<void>
+```
+
+This does nothing on some networks.
+On others, it opts into the given `contract`.
+
+---
+@{ref("js", "appOptedIn")}@{ref("js", "stdlib.accountAppOptedIn")}
+```js
+acc.appOptedIn(contract) => Promise<boolean>
```

Comment:
Clarify type as Contract / info

Reply:
This one would need some clarification for what type `contract` is. We need to ensure that the user understands it is the "info" for a contract (like the Reach type `Contract`) and not a contract handle in JS